src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -284,8 +284,18 @@ private void requireDockerApiVersionLessThan(final String required, final String
     assumeTrue(msg, dockerApiVersionLessThan(required));
   }
 
+  private boolean dockerApiVersionNot(final String expected) {
+    return compareVersion(dockerApiVersion, expected) != 0;
+  }
+
+  private void requireDockerApiVersionNot(final String version, final String msg) {
+    assumeTrue(msg, dockerApiVersionNot(version));
+  }
+
   @Test
   public void testSearchImage() throws Exception {
+    requireDockerApiVersionNot("1.19", "Docker 1.7.x sends the wrong Content-Type header for "
```

Comment:
@mattnworb Is there a way to handle the incorrect `Content-Type text/plain` header? I couldn't find it by googling jersey stuff.